flusher/flusher/db.py

```diff
@@ -139,6 +139,16 @@ def Column(*args, **kwargs):
     Column("balance", sa.String),
 )
 
+interchain_accounts = sa.Table(
+    "interchain_accounts",
+    metadata,
+    Column("id", sa.Integer, sa.Sequence("seq_interchain_account_id"), unique=True),
+    Column("address", sa.String, primary_key=True),
+    Column("connection_id", sa.String, sa.ForeignKey("connections.connection_id")),
+    Column("counterparty_port", sa.String),
+    Column("counterparty_address", sa.String),
+)
```

Comment:
This table should be an extension of accounts table? (ICA account should be in accounts table or not / or use account_id (primary_key + foreign key to account) instead of address?)

Reply:
already fixed to use account_id.